input-method.php

```diff
@@ -14,6 +15,13 @@
 ?>
 <Response>
     <Gather numDigits="1" timeout="10" action="input-method-result.php?SearchType=<?php echo $searchType ?>" method="GET">
+    
+    	<?php 
+			if ($playTitle == "1") {
+			echo "<Say voice=\"" . $voice . "\" language=\"" . $language . "\">" . $GLOBALS['title'] . "</Say>";
+			}
+		?>
+        
```

Comment:
This looks good.  The only thing would be to fix indentations.  Notice how I try to keep things lined up for readability.  https://github.com/radius314/yap/blob/master/meeting-search.php

Reply:
gotcha, thanks. im starting to get more of a grasp on this whole github, fork, push, pull, commit thing too I think.

Reply:
👍